integration-tests/tests/standard_cases/main.rs

```diff
@@ -1078,7 +1077,7 @@ pub fn test_access_key_reject_non_function_call(node: impl Node) {
         }),
     };
     let mut node_user = node.user();
-    let signer2 = InMemorySigner::from_random(AccountId::test_account(), KeyType::ED25519);
+    let signer2 = InMemorySigner::from_random("test".parse().unwrap(), KeyType::ED25519);
```

Comment:
unrelated to the PR, but it's not great that we have this many copies of the logic in the tests.